tritium-metrics/build.gradle

```
@@ -23,13 +23,14 @@ dependencies {
 
     testImplementation project(':tritium-test')
```

See if we can share a single `TestTaggedMetricRegistries`

```suggestion
    testImplementation project(':tritium-registry')
    testImplementation project(':tritium-test')
```

---

skipping as this ends up being nasty with gradle test dependencies, just going to leave the duplication for now